src/webdriver.nim

```diff
@@ -203,6 +213,126 @@ proc takeScreenshot*(self: Session): string =
 
   return respObj["value"].getStr()
 
+proc execute*(self: Session, code: string, args: varargs[JsonNode]): JsonNode =
+  let reqUrl = $(self.driver.url / "session" / self.id / "execute/sync")
+  let obj = %*{
+    "script": code,
+    "args": []
+  }
+  for arg in args:
+    obj["args"].elems.add arg
+
+  let resp = self.driver.client.post(reqUrl, $obj)
+  if resp.status != Http200:
+    raise newException(WebDriverException, resp.status)
+
+  let respObj = checkResponse(resp.body)
```

Comment:
I believe `checkResponse` will raise an exception so the line below will have no effect.

Reply:
Not sure I understand, this is still a problem. When the JSON has `error` in it, then `checkResponse` will raise because there will be no `value` key (https://github.com/dom96/webdriver/blob/1f0b47f4eb59fc285d8afcff736afd175621851f/src/webdriver.nim#L46). That's my assumption, maybe I'm wrong, but if not then the code below will never run.

Reply:
Error is inside the value key (https://github.com/jlipps/simple-wd-spec#execute-script). I forgot to put `respObj["value"]`.